timeseries/src/autogluon/timeseries/configs/presets_configs.py

```diff
@@ -12,22 +12,8 @@
             "num_trials": 20,
         },
     },
-    high_quality={
-        "hyperparameters": "default_hpo",
-        "hyperparameter_tune_kwargs": {
-            "scheduler": "local",
-            "searcher": "random",
-            "num_trials": 10,
-        },
-    },
-    good_quality={
-        "hyperparameters": "default_hpo",
-        "hyperparameter_tune_kwargs": {
-            "scheduler": "local",
-            "searcher": "random",
-            "num_trials": 2,
-        },
-    },
+    high_quality={"hyperparameters": "default"},
```

Comment:
Should we just drop these as the Predictor already seems to be handling the fallback?

Reply:
We notify the user inside the predictor, but I think the `@apply_presets` decorator will fail if we remove the presets completely
https://github.com/awslabs/autogluon/blob/bd1615584863e01b8123fb5d515e16cc5aca5639/timeseries/src/autogluon/timeseries/predictor.py#L183